bin/satellite-clone

```diff
@@ -1,98 +1,38 @@
-    parser.add_argument("--flush-cache",
-                        action="store_true",
-                        dest="flush_cache",
-                        default=False,
-                        help="clear the fact cache and run playbook")
-    parser.add_argument("-y",
-                        "--assume-yes",
-                        action="store_true",
-                        dest="assume_yes",
-                        default=False,
-                        help="Assume yes for each question")
-
-    args = parser.parse_args()
-
-    PRODUCTION_INSTALL_PATH = "/usr/share/satellite-clone"
-    DEFAULT_PLAYBOOK_FILE = os.path.join(PRODUCTION_INSTALL_PATH,
-                                         "satellite-clone-playbook.yml")
-
-    if not os.path.isdir(PRODUCTION_INSTALL_PATH):
-        print("It looks like satellite-clone has"
-              " not been installed properly, {0} does not exist.\n"
-              .format(PRODUCTION_INSTALL_PATH))
-        exit(1)
-
-    confirm = False
-    ansible_args = []
-    if args.step:
-        ansible_args.append("--step")
-    if args.list_tasks:
-        ansible_args = []
-        ansible_args.append("--list-tasks")
-        confirm = True
-    if args.flush_cache:
-        ansible_args.append("--flush-cache")
-    if args.assume_yes:
-        confirm = True
-
-    if not confirm:
-        yes_or_no("This will initiate satellite-clone. "
-                  "Do you want to proceed?")
-
-    os.environ["ANSIBLE_CONFIG"] = os.path.join(PRODUCTION_INSTALL_PATH,
-                                                'ansible.production.cfg')
-
-    # We need to import here because the ANSIBLE_CONFIG environment variable is
-    # checked at import time (instead of when calling PlaybookCLI)
-    from ansible.cli.playbook import PlaybookCLI
-
-    inventory_path = os.path.join(PRODUCTION_INSTALL_PATH, 'inventory')
-    ansible_playbook = (["ansible-playbook", "-i", inventory_path] +
-                        ansible_args +
-                        [DEFAULT_PLAYBOOK_FILE])
-
-    print(" ".join(ansible_playbook))
-
-    cli = PlaybookCLI(ansible_playbook)
-    cli.parse()
-    exit_code = cli.run()
-    sys.exit(exit_code)
-
-
-if __name__ == '__main__':
-    main()
+#!/bin/bash
+
+PRODUCTION_INSTALL_PATH="/usr/share/satellite-clone"
+DEFAULT_PLAYBOOK_FILE="${PRODUCTION_INSTALL_PATH}/satellite-clone-playbook.yml"
+INVENTORY_PATH="${PRODUCTION_INSTALL_PATH}/inventory"
+ANSIBLE_CONFIG="${PRODUCTION_INSTALL_PATH}/ansible.production.cfg"
+export ANSIBLE_CONFIG
+
+usage() {
+  echo "usage: satellite-clone [-h] [--step] [--list-tasks] [--flush-cache] [-y]
+
+options:
+  -h, --help        show this help message and exit
+  --step            interactive: confirm each task before running
+  --list-tasks      list tasks that will be run in the playbook
+  --flush-cache     clear the fact cache and run playbook
+  -y, --assume-yes  Assume yes for each question"
+}
+
+if [[ "$@" == "--help" ]] || [[ "$@" == "-h" ]]; then
```

Comment:
Does this work to check if an entry is in an array? If so, I learned something new.

Reply:
it doesn't :)